app/models/order.rb

```diff
@@ -42,4 +44,14 @@ def finalize_order
     end
     save!
   end
+
+  def save
+    self.owner = ManageIQ::API::Common::Request.current.user.username
+    super
+  end
+
+  def save!
+    self.owner = ManageIQ::API::Common::Request.current.user.username
+    super
+  end
```

Comment:
Is this just `before_create` logic?  Same goes for the other models.  Possible mixin?

Reply:
@Fryguy, callbacks make sense here, but was wondering where you are at with them being used in the microservices.  I seemed to remember there was a request to stay away from them or minimize their use?